libraries/AP_HAL/board/chibios.h

```diff
@@ -175,3 +175,9 @@
 #ifndef AP_BOARDCONFIG_MCU_MEMPROTECT_ENABLED
 #define AP_BOARDCONFIG_MCU_MEMPROTECT_ENABLED 0
 #endif  // AP_BOARDCONFIG_MCU_MEMPROTECT_ENABLED
+
+#if defined(STM32H7)
```

Comment:
also check for at least 1M ram, so we don't impact H723

Reply:
Fixed